python/viewer/radar_data_streamer.py

```diff
@@ -29,10 +30,14 @@ def __init__(self, data_stream_path, proto_type, python_type, header_size=8):
         self.proto_type = proto_type
         self.python_type = python_type
         self.header_size = header_size
-        self.data_streams = None
+        self.mode = mode
+        self.data_stream = None
 
     def __enter__(self):
-        self.data_stream = open(self.data_stream_path, "rb")
+        self.data_stream = open(self.data_stream_path, self.mode)
+        if self.mode == 'wb':
+            # write 8 bytes header
+            self.data_stream.write(b"\x41\x48\x41\x56\x00\x00\x00\x00")
```

Comment:
does this string of bytes actually mean anything? Is that better than just pack it with all 0s?

Reply:
For consistency I used what we use in` io.py` in the tracker pipeline.